### ISSUE TITLE
All of the directories in the array doesn't exist. Update emmet.extensionsPath setting

### ISSUE THREAD

**Issue by JacksonKearl** [microsoft/vscode#118008]
I have not configured this setting, so I'd expect to not see this error.

(Note this is only building from sources, disregard if you've already fixed)

![image](https://user-images.githubusercontent.com/8586769/109750028-8ada4700-7b90-11eb-8f08-be1c8f2afe67.png)

**Comment by rzhao271** [comment]
Ok. I looked into the problem with @lancewl and we have concluded the following:

1. This issue only affects OSS builds on the main branch, and therefore is not a candidate for the release/1.54 branch.
2. This issue occurs due to a tragic accident between a new change in this repo and vscode-emmet-helper which worked together to make it so that the notification shows up not only when there's an actual issue with the extensionsPath, but also when the user didn't specify any.

I will push in a mitigation fix to the main branch. The actual fix will be out later.
